plugin/build_baron.go

```diff
@@ -102,9 +102,13 @@ func (bbp *BuildBaronPlugin) GetPanelConfig() (*PanelConfig, error) {
 					template.HTML(`<script type="text/javascript" src="/static/plugins/buildbaron/js/task_build_baron.js"></script>`),
 				},
 				DataFunc: func(context UIContext) (interface{}, error) {
-					enabled := len(bbp.opts.Projects[context.ProjectRef.Id].TicketSearchProjects) > 0
+					bbSettings, ok := BbGetProject(evergreen.GetEnvironment().Settings(), context.ProjectRef.Id, "")
+					if !ok {
+						return nil, nil
+					}
+					enabled := len(bbSettings.TicketSearchProjects) > 0
 					if !enabled {
-						enabled = len(bbp.opts.Projects[context.ProjectRef.Identifier].TicketSearchProjects) > 0
+						enabled = len(bbSettings.TicketSearchProjects) > 0
```

Comment:
Looks like we don't need this if statement anymore, since we're setting it to the same thing as 109